appsec-kit-v7/src/main/java/com/vaadin/appsec/v7/service/AppSecKitInitializer.java

```diff
@@ -40,6 +40,8 @@ public class AppSecKitInitializer implements HttpSessionListener {
 
     private static final List<String> initializedVaadinServiceNames = new CopyOnWriteArrayList<>();
```

Comment:
`static` could be removed.

Reply:
That's it `HttpSessionListener` instances are singletons, I'm looking for confirmation about that.

Reply:
Looks like that, multiple sessions are handled by the same listener instance.

Reply:
Yeah, I meant only that the `static` keyword could be removed.